Userland/Libraries/LibAudio/FlacWriter.h

```diff
@@ -83,6 +87,12 @@ class FlacWriter : public Encoder {
     ErrorOr<void> set_sample_rate(u32 sample_rate);
     ErrorOr<void> set_bits_per_sample(u16 bits_per_sample);
 
+    // The FLAC encoder by default tries to reserve some space for seek points,
+    // but that may not be enough if more than approximately four minutes of audio are stored.
```

Comment:
How do we need to deal with situations where `sample_count_hint` is never invoked and we pass the `default_padding` size?

Reply:
I forgot to make the code just not write superflous seek points, which is the fix here. Oops.